content/influxdb/v2.0/visualize-data/visualization-types/band.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Band visualization
+list_title: Band
+list_image: /img/influxdb/2-0-visualizations-Band-example.png
+description: >
+  The Band view displays the single value most recent value for a time series in a Band view.
+weight: 206
+menu:
+  influxdb_2_0:
+    name: band
+    parent: Visualization types
+---
+
+The **Band** visualization displays the upper and lower boundaries for groups of data over time.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-Band-example.png" alt="Band example" />}}
+
+Select the **Band** option from the visualization dropdown in the upper left.
+
+## Band behavior
+
+The band visualization displays
+
+## Band controls
+
+To view **Band** controls, click **{{< icon "gear" >}} Customize** next to the visualization dropdown.
+
+
+###### Data
+- **X Column**: Select a column to display on the x-axis.
+- **Y Column**: Select a column to display on the y-axis.
+- **Time Format**: Select the time format. Options include:
+    - `MM/DD/YYYY HH:mm:ss` (default)
+    - `MM/DD/YYYY HH:mm:ss.SSS`
```

Comment:
curious do we define SSS elsewhere?

Reply:
We don't. This is in a handful of other visualization types too.